compiler/quilt/tools/store.py

```diff
@@ -315,15 +325,21 @@ def parse_package_extended(name):
                 else:
                     # usr/pkg:hashval
                     hash = versioninfo
-        owner, pkg, subpath = parse_package(name, allow_subpath=True)
+        team, owner, pkg, subpath = parse_package(name, allow_subpath=True)
     except ValueError:
         pkg_format = 'owner/package_name/path[:v:<version> or :t:tag or :h:hash]'
         raise CommandException("Specify package as %s." % pkg_format)
     return owner, pkg, subpath, hash, version, tag
 
 def parse_package(name, allow_subpath=False):
     try:
-        values = name.split('/')
+        values = name.split(':', 1)
+        if len(values) > 1:
+            team = values[0]
+        else:
+            team = None
```

Comment:
BUG?  len() > 0 ?

also:
team = values[0] if len(values) > 1 else None

Reply:
not a bug, check how split works. Will fix syntax suggestion